### PR TITLE
Added SortWithSwapsPlugin.java

### PR DESCRIPTION
**Pull request by esther-j**
Added SortWithSwapsPlugin.java from https://drive.google.com/file/d/0B-3-IFNZtYAPTFUyNmxkckxROWc/view

### CODE REVIEW

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
+package edu.cmu.cs.cs214.rec10.plugin;
+
+import edu.cmu.cs.cs214.rec10.framework.core.GameFramework;
+import edu.cmu.cs.cs214.rec10.framework.core.GamePlugin;
+
+import java.util.Random;
+
+public class SortWithSwapsPlugin implements GamePlugin {
```
Comment (yuruiw) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java:9]: Please add a new comment for this class.

**Review by yuruiw — CHANGES_REQUESTED** [review]
Please add a new comment

**Comment by esther-j** [comment]
Newly added comment to the class header

**Review by yuruiw — APPROVED** [review]
Good job!

**Pull request merged**